### DISCUSSION TITLE
combat contre bot de test

### DISCUSSION THREAD

**Discussion by ghosts-arch** [Crownicles/Crownicles#1709] [Nouvelles idées]
le but serait de pouvoir combattre sur 5 tours contre un mannequin (qui se regene à chaque tour) qui as des stats et une classe aléatoire pour pouvoir tester une classe ou un équipement

**Comment by ghosts-arch** [comment]
des attaques aussi j'ai oublié

**Comment by BastLast** [comment]
Jai peur que les gens fassent que ca a la place des fight amicaux

**Comment by ghosts-arch** [comment]
ca rapporterait rien
et ca serait validé par aucune quete

**Reply by BastLast** [reply]
Jai dit amicaux

**Reply by ghosts-arch** [reply]
j'ai oublié une partie encore

**Reply by ghosts-arch** [reply]
le but c'est juste d'avoir un environnement de test pour comprendre comment fonctionne une classe, une attaque et créer une strat, le tout contre différentes classes et niveaux sans devoir trouver une personne du même niveau que toi et la classe contre la quelle tu veux strater

**Reply by BastLast** [reply]
Je pense qu'une grosse partie du fun est d'apprendre en jouant

**Reply by ghosts-arch** [reply]
c'est juste long, et pour juste tester un équipement ou une attaque c'est pas cool parce que tu dépends de quelqu'un aussi, si y'as perso de disponible ca risque d'être long, surtout en dehors du serveur officiel
